src/style.css

```
@@ -0,0 +1,317 @@
+@font-face {
+  font-family: myFont;
+  src: url(Crucial-Medium.otf);
+}
+@font-face {
+  font-family: secondFont;
+  src: url(BAD\ GRUNGE.ttf);
```

Poți folosi ghilimele, ca să nu folosești escape.
```suggestion
  src: url("BAD GRUNGE.ttf");
```